app/plugin_test.go

```diff
@@ -537,7 +537,7 @@ func TestPluginSync(t *testing.T) {
 			pluginFileReader, err := os.Open(filepath.Join(path, "testplugin.tar.gz"))
 			require.NoError(t, err)
 			defer pluginFileReader.Close()
-			_, appErr = th.App.WriteFile(pluginFileReader, th.App.getBundleStorePath("testplugin.tar.gz"))
```

Comment:
This test was spuriously writing `testplugin.tar.gz.tar.gz`, so I've fixed that. The existing `TestPluginSync` verifies the new behaviour.